keras_nlp/models/bert.py

```diff
@@ -782,41 +734,50 @@ def BertLarge(weights=None, vocabulary_size=None, name=None, trainable=True):
     return model
 
 
+arch_classes = {
```

Comment:
This feels hard to read name everywhere it's imported. Seeing just the import I have no idea what an arch_class is.

Can we just inline this in a function? `lookup_model_class(name)` or whatever we want the name to be. Could also do this as a follow up.